src/main/java/net/snowflake/ingest/utils/Utils.java

```diff
@@ -430,4 +430,18 @@ public static String concatDotPath(String... path) {
     }
     return sb.toString();
   }
+
+  /**
+   * Get the little endian int from a byte array with offset
+   *
+   * @param bytes the byte array
+   * @param offset the offset
+   * @return the little endian int
+   */
+  public static int getLittleEndianInt(byte[] bytes, int offset) {
```

Comment:
(dupe) remove as we'll use parquet library's BytesUtils

Reply:
Resolved.